utility/utils.py

```diff
@@ -8,15 +8,20 @@
 import subprocess
 import time
 import traceback
-
 import requests
 import yaml
+import re
+import smtplib
 
 from ocs.exceptions import (
     CommandFailed, UnsupportedOSType, TimeoutExpiredError,
 )
 from ocsci import config
 from .aws import AWS
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from ocs import constants
+from bs4 import BeautifulSoup
```

Comment:
Where do you get the `bs4` module? I don't see it in the requirements...

Reply:
Sorry I've missed it in the `setup.py`, as tox is failing on my laptop... but it seems to be some different problem.